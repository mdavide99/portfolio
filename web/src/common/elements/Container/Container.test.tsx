import { Container } from "./Container";
import { render } from "@testing-library/react";

describe("Seo", () => {
  it("renders Seo correctly", () => {
    const { container } = render(<Container>TEST</Container>);
    expect(container).toMatchSnapshot();
  });
});
