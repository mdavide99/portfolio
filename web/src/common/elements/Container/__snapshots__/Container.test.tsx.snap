// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Seo renders Seo correctly 1`] = `
<div>
  <div
    class="container mx-auto px-4"
  >
    TEST
  </div>
</div>
`;
